File: app/src/main/java/fr/paug/androidmakers/ui/activity/BaseActivity.java
package fr.paug.androidmakers.ui.activity;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import fr.paug.androidmakers.util.ThemeUtils;

public abstract class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        ThemeUtils.ensureRuntimeTheme(this);
        super.onCreate(savedInstanceState);
    }

}

File: app/src/main/java/fr/paug/androidmakers/ui/activity/BaseActivity.kt
package fr.paug.androidmakers.ui.activity

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import fr.paug.androidmakers.model.RoomKt
import fr.paug.androidmakers.model.SessionKt
import fr.paug.androidmakers.util.ScheduleSessionHelper

import fr.paug.androidmakers.util.ThemeUtils
import io.openfeedback.android.model.Project

abstract class BaseActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        ThemeUtils.ensureRuntimeTheme(this)
        super.onCreate(savedInstanceState)
    }

    override fun onDestroy() {
        super.onDestroy()
        ScheduleSessionHelper.cancelToast()
    }
}